lib/rgeo/cartesian/factory.rb

```diff
@@ -183,8 +186,8 @@ def parse_wkb(str)
 
       # See RGeo::Feature::Factory#point
 
-      def point(x, y, *extra)
-        PointImpl.new(self, x, y, *extra)
+      def point(x_coord, y_coord, *extra)
```

Comment:
@BuonOmo to me that makes sense, but wasn't sure if you guys wanted to make that allowance or not because you could then use `x`, `y`, or `z` outside the context of coordinates. Either way, doesn't matter to me 👍

Reply:
@BuonOmo yeah I agree we can make an exception to that check since `x` `y` `z` and `m` are meaningful in this context.

Reply:
@BuonOmo would you like me to revert all changes I made like this then or just this one method?

Reply:
@thestelz I'd go for all, otherwise the difference will stay forever!